test/integration/packer_pipeline_node/inspec/packer_pipeline_node_spec.rb

```diff
@@ -51,7 +51,7 @@
   it { should be_executable }
 end
 
-describe gem('openstack_taster', "/opt/#{chef}/embedded/bin/gem") do
+describe gem('openstack_taster', '/opt/${chef}/embedded/bin/gem') do
```

Comment:
This is incorrect. Why was this changed?

Reply:
oh shoot, it got flagged by cookstyle for some reason

Reply:
oh , what happened I think is that the $ was in my version, but then on rebase it changed and i applied the single quotes in the merge